roles/confluent.common/tasks/main.yml

```
@@ -16,14 +16,14 @@
     - ansible_distribution
 
 # JVM Installation and Repo Setup
-- include_tasks: redhat.yml
-  when: ansible_os_family == "RedHat"
 
-- include_tasks: ubuntu.yml
-  when: ansible_distribution == "Ubuntu"
+- include_tasks: "{{ ansible_distribution }}.yml"
```

I wonder if you could do:

```suggestion
- include_tasks: "{{ ansible_os_family if  ansible_os_family == 'RedHat' else ansible_distribution}}.yml"
```

And then we dont need the Centos.yml file